### PR TITLE
make output_base directory configurable

### PR DESCRIPTION
**Pull request by sheldonneuberger-sc**
Adds an outputBase string configuration that changes where output_base is placed. This is convenient for when someone wants bazel using something other than TMPDIR. This is useful if you use a ramdisk for faster builds, or if your TMPDIR has space or security constraints.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -116,6 +116,11 @@
                     "default": false,
                     "description": "Use the same Bazel server for queries and builds. By default, vscode-bazel uses a separate server for queries so that they can be executed in parallel with builds. You can enable this setting if running multiple Bazel servers has a negative performance impact on your system, but you may experience degraded performance in Visual Studio Code for operations that require queries."
                 },
+                "bazel.outputBase": {
```
Comment (jfirebaugh) [inline package.json:119]:
```suggestion
                "bazel.queryOutputBase": {
```

Just to make explicit the scope of this setting.

**Review by jfirebaugh — COMMENTED** [review]
Thanks @sheldonneuberger-sc!

Are you able to sign the CLA?

**Comment by sheldonneuberger-sc** [comment]
> Are you able to sign the CLA?
Yes, we (snapchat) have signed it before but last I checked we lost the details of it. Might take a few days to figure it out again.

**Comment by sheldonneuberger-sc** [comment]
Got myself onto the CLA list, I believe this can be merged now.

**Comment by jfirebaugh** [comment]
Looks like now CI is complaining about a code style issue. This line is exeeding 80 characters (sorry we don't have autoformatting set up yet):

https://github.com/bazelbuild/vscode-bazel/blob/076e3432f306f5fd44a93a021cfea4c6a5b87ba9/src/bazel/bazel_query.ts#L132

**Review by jfirebaugh — APPROVED** [review]
Thanks!

**Pull request merged**